.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -5,20 +5,18 @@ about: This issue is used to track a feature/project that may span several days
   and manage work.
 title: "[Epic] Title"
 labels: ".Epic"
-
 ---
 
-**Links**
-- product doc: _link to product doc_
-- eng doc: _link to technical design doc, if any_
-- feature branch: `branch-name` _this should be the feature branch where this work will be done in. PRs will be delivered against this branch_
-- issue links: _related issues if any_
-
-**Implementation Plan**
-
+### Links
 
-***Milestone 1***
-_insert tasklist here_
+- [product doc](LINK_TO_PRODUCT_DOC)
+- [engineering doc](LINK_TO_ENGINEERING_DOC)
```

Comment:
above it says "link to technical design doc" - should this be LINK_TO_TECHNICAL_DESIGN_DOC?